lib/index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-module.exports = require('./linter');
+module.exports = require('./linter').Linter;
```

Comment:
The lines just below still mutate the class.

It isn't a big deal though, I would just like to fix it in the long run. Fine for a follow up...